conductor-client/src/main/kotlin/com/openlattice/data/storage/redshift/RedshiftDataTables.kt

```diff
@@ -0,0 +1,221 @@
+        //If you change this such that columns generated are sensitive to being discarded make sure you update logic.
+        private val dataColumns = supportedEdmPrimitiveTypeKinds
+                .map(PostgresEdmTypeConverter::map)
+                .associateWith { nonIndexedValueColumn(it) to btreeIndexedValueColumn(it) }
+
+        private val nonIndexedColumns = dataColumns.map { it.value.first }
+        private val btreeIndexedColumns = dataColumns.map { it.value.second }
+
+        val dataTableMetadataColumns = listOf(
+                ENTITY_SET_ID_STRING,
+                ID_STRING,
+                ORIGIN_ID_STRING,
+                PROPERTY_TYPE_ID_STRING,
+                HASH_STRING,
+                LAST_WRITE,
+                LAST_PROPAGATE,
+                LAST_TRANSPORT,
+                VERSION,
+                VERSIONS
+        )
+        
+        val dataTableValueColumns = nonIndexedColumns
+        val dataTableColumns = dataTableMetadataColumns + dataTableValueColumns
+
+        private val columnDefinitionCache = CacheBuilder.newBuilder().build(
+                object : CacheLoader<Pair<IndexType, EdmPrimitiveTypeKind>, PostgresColumnDefinition>() {
+                    override fun load(key: Pair<IndexType, EdmPrimitiveTypeKind>): PostgresColumnDefinition {
+                        return buildColumnDefinition(key)
+                    }
+
+                    override fun loadAll(
+                            keys: MutableIterable<Pair<IndexType, EdmPrimitiveTypeKind>>
+                    ): MutableMap<Pair<IndexType, EdmPrimitiveTypeKind>, PostgresColumnDefinition> {
+                        return keys.associateWith { buildColumnDefinition(it) }.toMutableMap()
+                    }
+
+                    private fun buildColumnDefinition(
+                            key: Pair<IndexType, EdmPrimitiveTypeKind>
+                    ): PostgresColumnDefinition {
+                        val (indexType, edmType) = key
+                        return when (indexType) {
+                            IndexType.BTREE -> btreeIndexedValueColumn(
+                                    PostgresEdmTypeConverter.map(edmType)
+                            )
+                            IndexType.NONE -> nonIndexedValueColumn(
+                                    PostgresEdmTypeConverter.map(edmType)
+                            )
+                            else -> throw IllegalArgumentException(
+                                    "HASH or GIN indexes are not yet supported by openlattice."
+                            )
+                        }
+                    }
+                }
+        )
+
+        @JvmStatic
+        fun buildDataTableDefinition(): PostgresTableDefinition {
+            val columns = dataTableColumns.toTypedArray()
+
+            val tableDefinition = CitusDistributedTableDefinition("data")
```

Comment:
Since we're no longer using Citus, can't you use `PostgresTableDefinition` here instead (and remove `.distributionColumn(PARTITION)` below)?